### ISSUE TITLE
Page number in powerpoint

### ISSUE THREAD

**Issue by tobbylefou** [rstudio/rmarkdown-book#28]
Hello,
Manu thanks for your very interesting book. 
I have a question concerning chapter "4.4 PowerPoint presentation" :
How can I get the page number be displayed on the generated slides ?
In advance many thanks
Best regards

**Comment by mikey-harper** [comment]
Hi  @tobbylefou , have you tried creating a custom `.pppt` template as outlined in the Section [4.4.1](https://bookdown.org/yihui/rmarkdown/powerpoint-presentation.html#ppt-templates)?

**Comment by tobbylefou** [comment]
Thank you very much Michael for your very quick reply.

I finally managed to get the slide number. To do so, in the slide master of my reference pptx document:
- I deleted the default text box in the footer dedicated to slide number
- And I created a new text box from scratch, that includes the slide number

I close the issue
I note you tip, concerning the usage of Stackoverflow instead of "Issue" for asking questions.

Kind regards
